test/commands/git/sync_track_test.rb

```diff
@@ -27,6 +27,41 @@ class Git::SyncTrackTest < ActiveSupport::TestCase
     assert_equal git_track.head_sha, track.synced_to_git_sha
   end
 
+  test "git sync SHA does not change when concept syncing fails" do
+    track = create :track, slug: 'fsharp', active: true, synced_to_git_sha: "98403713252d41babae8353793ea5ec9ad7d770f"
+    Git::SyncConcept.expects(:call).raises(RuntimeError)
+
+    suppress(RuntimeError) do
```

Comment:
I found this somewhere. Let me know if this is the right mechanism to ignore an error

Reply:
Never come across `suppress` before. It's part of Rails FYI: https://api.rubyonrails.org/classes/Kernel.html#method-i-suppress

I normally use `assert_raises` to suppress, as that has the side effect of also ensuring the raising actually happens and blows the block. So I'd probably do that here too.